sdk/search/search-documents/src/serviceUtils.ts

```diff
@@ -79,12 +87,49 @@ export function convertCognitiveServicesAccountToPublic(
   }
 }
 
-export function convertAnalyzersToGenerated(analyzers?: Analyzer[]): AnalyzerUnion[] | undefined {
+export function convertTokenFiltersToGenerated(
+  tokenFilters?: TokenFilter[]
+): TokenFilterUnion[] | undefined {
+  if (!tokenFilters) {
+    return tokenFilters;
+  }
+
+  const result: TokenFilterUnion[] = [];
+  for (const filter of tokenFilters) {
+    switch (filter.odatatype) {
+      case "#Microsoft.Azure.Search.NGramTokenFilter":
```

Comment:
so the idea is if the customer tries to create something with V1 we'll convert it to V2? 

How does this work if the user wants to update an existing V1 filter that is defined in their service?

Reply:
Per the discussion in the meeting, the requirement is:

1. When the user wants to send something, always override it with V2. 
2. If the server sends V1, allow it to pass it through. If the user modifies something in it and sends it back (as in createOrUpdate) override it with V2

Reply:
Ok. I synched up again with Heath. Based on the conversation, we need to allow it. So, removing all the overrides here.

Reply:
Done